lib/config/presets/internal/workarounds.ts

```diff
@@ -13,6 +13,7 @@ export const presets: Record<string, Preset> = {
       'workarounds:typesNodeVersioning',
       'workarounds:reduceRepologyServerLoad',
       'workarounds:doNotUpgradeFromAlpineStableToEdge',
+      'workarounds:ignoreDotnet7Preview',
```

Comment:
Added to workarounds list, because I can't see how it can be useful otherwise

Reply:
that's what I want to avoid, as some users might want those updates?